cpp/src/stream_compaction/distinct_count.cu

```diff
@@ -34,6 +34,8 @@
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/exec_policy.hpp>
 
+#include <cuco/static_set.cuh>
```

Comment:
I expect to see some header removed from this?

Reply:
Not for now, some common utilities will no longer be needed but I will leave it to the `distinct` refactoring with `cuco::static_reduction_map` to avoid back-and-forth effort.